src/v2/view-builder/views/TerminalView.js

```diff
@@ -9,6 +11,16 @@ const Body = BaseForm.extend({
     this.$el.addClass('terminal-state');
   },
 
+  title () {
+    const messagesObjs = this.options.appState.get('messages');
+    if (messagesObjs && Array.isArray(messagesObjs.value)) {
+      if (messagesObjs.value.some(messagesObj => messagesObj.i18n?.key === 'idx.return.link.expired')) {
+        return 'Verify with your email';
```

Comment:
create an i18n key instead of hard code English.

Reply:
with is the translation process? or we can keep only english for now?

Reply:
seems I can reuse exist one
```
oie.email.mfa.title = Verify with your email
```

Reply:
hm.. `oie.email.mfa.title` is intent for email factor page.
why you need to rename the title?

Reply:
@VadymYaremchuk-okta - When you add a new entry to `login.properties`, it'll be picked up by our i18n tooling to request translations through our l10n vendor. Most of this is automated - so the action for devs is pretty straightforward.

Also, instead of reusing an existing property key - I'd suggest creating a new key that better matches with this view. For example `oie.return.link.expired.title`.

Reply:
Also - is this text settled? I'm not sure an enduser will understand what we're asking them to verify in this flow.

Reply:
Sure, can create new i18n key if needed.

The title I took according the screenshot Vince provided me with https://oktawiki.atlassian.net/wiki/spaces/eng/pages/1281525702/OIE-AppEmbed-+POD+Deployment+Models+Wiki?preview=/1281525702/1345359732/image%20(7).png

Before the title was `Authenticate` (default one). I can keep the old one.

The error message itself is controlled by backend. 

What is the process of i18n translation?

Reply:
Full process is described [here](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/741293778/i18n+dev+process+From+Oct+2019+onwards).

Reply:
seems to simplify this we can keep the currently exist title `Authenticate`.
@vincentvoong-okta @wdawson can you please confirm that it is OK to keep the exist `Authenticate` title for terminal view in case of expired email link used?

Reply:
@VadymYaremchuk-okta can you include a screenshot for the page you're talking about? I want to make sure I understand the context.

Reply:
@wdawson 
screen with the issue:
<img width="535" alt="Screen Shot 2020-10-07 at 4 07 01 PM" src="https://user-images.githubusercontent.com/44722868/95397143-53a08c00-08b7-11eb-9fc1-dbc203fb80ea.png">
screen with the fix based on UI mock:
<img width="468" alt="Screen Shot 2020-10-07 at 4 07 47 PM" src="https://user-images.githubusercontent.com/44722868/95397161-5f8c4e00-08b7-11eb-8488-4f2cef6008f9.png">

Reply:
UI mock received from Vince:
![image (7)](https://user-images.githubusercontent.com/44722868/95397636-944cd500-08b8-11eb-9b4d-76e64c411fb9.png)

Reply:
Thanks, sounds good.